pkg/resource/sensitive.go

```diff
@@ -241,7 +244,9 @@ func GetSensitiveParameters(ctx context.Context, client SecretClient, from runti
 				var sensitives []interface{}
 				for _, s := range *sel {
 					sensitive, err = client.GetSecretValue(ctx, s)
-					if err != nil {
+					if err != nil && kerrors.IsNotFound(err) {
```

Comment:
same nitpick as above for better readability.

Reply:
done